presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -46,6 +46,8 @@
     private final StreamLayout streamLayout;
     private final boolean integerDictionaryEncodingEnabled;
     private final boolean stringDictionarySortingEnabled;
+    // TODO: Remove flag ignoreDictionaryRowGroupSizes once memory is handled for dictionary rowGroup sizes.
```

Comment:
Can we make the comment something like, 

Originally the dictionary row group sizes were not included in memory accounting due to a bug.
But fixing the bug caused the query to OOM. This flag is a hack to maintain the hacky behavior
while the query OOMs are addressed. Remove this flag after the query OOMs are fixed.

Reply:
Done